frontend/appflowy_flutter/lib/plugins/database/widgets/row/row_banner.dart

```diff
@@ -281,6 +289,11 @@ class RowActionButton extends StatelessWidget {
 }
 
 class _TitleSkin extends IEditableTextCellSkin {
+  _TitleSkin({this.emojiWidget});
+
+  final Widget? emojiWidget;
+  ui.TextDirection? lastDirection;
+
   @override
   Widget build(
     BuildContext context,
```

Comment:
I'm not sure I like this.

Auto bidi is okay for text fields, but if we want to reverse the positioning of widgets in a row, the user should choose RTL from the settings. Otherwise, there are too many changes that we need to do throughout the entire application.